ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e.yaml

```diff
@@ -184,7 +220,9 @@ objects:
           exit 0
         }
 
-        ${TEST_COMMAND}
+        #FIXME: hard wire this to run parallel tests
+        #${TEST_COMMAND}
+        TEST_SUITE=openshift/conformance/parallel run-tests
```

Comment:
Doesn't seem like part of this PR.  Can it move into a separate PR if you feel like there's a reason we can't delegate the choice of tests to the job definition?

Reply:
See my note above. My intent was to remove this once I see it pass. The issue is the OpenStack test jobs have TEST_COMMAND set to something that runs only a minimal set of smoke tests. This needs to change over time I think but as is now I just had this FIXME here to test it real quick in CI.